ui/src/app/applications/components/application-deployment-history/application-deployment-history.tsx

```diff
@@ -31,16 +33,18 @@ export const ApplicationDeploymentHistory = ({
             {recentDeployments.map((info, index) => (
                 <div className='row application-deployment-history__item' key={info.deployedAt} onClick={() => selectDeployment(index)}>
                     <div className='columns small-3'>
-                        <i className='fa fa-clock'/>
-                        <div>{info.deployedAt} - {info.nextDeployedAt || 'now'}</div>
+                        <div>
+                            <i className='fa fa-clock'/> <Moment fromNow={true}>{info.deployedAt}</Moment>
+                        </div>
+                        <div>(<Moment local={true}>{info.deployedAt}</Moment>)</div>
```

Comment:
There was a request to switch between UTC and local time zone. Please create a wrapper around Moment. As a next step we can enhance wrapper component to use user preferences to switch between time zones.

Reply:
✔️